quiz.html

```diff
@@ -1,58 +1,84 @@
 <html lang="en">
   <head>
     <title>Quiz</title>
-    <link rel="stylesheet" href="/style.css" >
+    <link rel="stylesheet" href="/style.css" />
     <script src="/script.js" defer></script>
   </head>
-<body>
-  <header>
-    <img src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Flogo.png?v=1605558587857" alt="logo">
-    <nav>
+  <body>
+    <header>
+      <img
+        class="logo"
+        src="https://cdn.glitch.com/2d266749-7995-4116-89e7-b22714ac7a5b%2Flogo.png?v=1605558587857"
+      />
+      <nav class="Project">
         <a href="Mobile.html"> Home </a>
         <a href="project.html"> Project </a>
-    </nav>
-  </header>
-<h1 class="quiz-title">
-  Quiz
-</h1>
-<p class="quiz-description">
-  You just explored four planets but Rose wasn’t on either of them. To guide you through your journey Fox is willing to help you. Answer to these questions to unlock him. 
-</p>
-  
-<form action="/action_page.php">
-  <p class="questions">1-How many moons does Mars have</p>
-  <input type="radio" id="2" name="number" value="2">
-  <label for="2">2</label><br>
-  <input type="radio" id="1" name="number" value="1">
-  <label for="1">1</label><br>
-  <input type="radio" id="50" name="number" value="50">
-  <label for="50">50</label><br>  
-  <input type="radio" id="13" name="number" value="13">
-  <label for="13">13</label><br>  
+      </nav>
+    </header>
+    <h2 class="quiz-title">
+      Quiz
+    </h2>
+    <p class="Description-quiz">
+      You just explored four planets but Rose wasn’t on either of them. To guide
+      you through your journey Fox is willing to help you. Answer to these
+      questions to unlock him.
+    </p>
 
-  <p class="question"> 2- What percentage of the earth surface is covered with water?</p>
-  <input type="radio" id="percent1" name="percent" value="50%">
-  <label for="percent1">50%</label><br>
-  <input type="radio" id="percent2" name="percent" value="60%">
-  <label for="percent1">60%</label><br> 
-  <input type="radio" id="percent3" name="percent" value="70%">
-  <label for="percent1">70%</label><br>
-  <input type="radio" id="percent4" name="percent" value="80%">
-  <label for="percent1">80%</label><br>
-  
-  <p class="question"> 3- What is the largest planet in the solar system?</p>
-  <input type="radio" id="planet1" name="planet" value="Mars">
-  <label for="planet1">Mars</label><br>
-  <input type="radio" id="planet2" name="planet" value="Venus">
-  <label for="planet1">Venus</label><br>
-  <input type="radio" id="planet3" name="planet" value="Jupiter">
-  <label for="planet1">Jupiter</label><br>
-  <input type="radio" id="planet4" name="planet" value="Saturn">
-  <label for="planet1">Saturn</label><br>
-  
-  <input type="submit" value="Submit">
-</form>  
-  
+    <form action="/action_page.php">
```

Comment:
I don't think you actually need the `action` attribute here - you don't have an `action_page.php` to send this form data to. This attribute is only needed if you're actually sending data to a server somewhere. A good example would be a contact form, where you're saving someone's information to a database. Since you're not saving any of this question info, you can skip it!